tests/test_image_processor.py

```diff
@@ -0,0 +1,63 @@
+# coding=utf-8
```

Comment:
great tests!

Reply:
Could we now also add all the combinations of tests? E..g decode(encode(PT), output_type={np, PIL}) == convert_to_{np, PIL}(PT))

Think this can be handled quite nicely in a loop :-)